numba_dpex/dpctl_iface/usm_shared_allocator_ext.c

```diff
@@ -30,17 +30,17 @@
 #include <stdbool.h>
 
 #if defined __has_include
-#  if __has_include(<dpctl_sycl_interface.h>)
-#    include <dpctl_sycl_interface.h>
-#  else
-#    include <dpctl_sycl_queue_interface.h>
-#    include <dpctl_sycl_queue_manager.h>
-#    include <dpctl_sycl_usm_interface.h>
-#  endif
+#if __has_include(<dpctl_sycl_interface.h>)
+#include <dpctl_sycl_interface.h>
 #else
-#  include <dpctl_sycl_queue_interface.h>
-#  include <dpctl_sycl_queue_manager.h>
-#  include <dpctl_sycl_usm_interface.h>
+#include <dpctl_sycl_queue_interface.h>
+#include <dpctl_sycl_queue_manager.h>
+#include <dpctl_sycl_usm_interface.h>
+#endif
+#else
+#include <dpctl_sycl_queue_interface.h>
+#include <dpctl_sycl_queue_manager.h>
+#include <dpctl_sycl_usm_interface.h>
```

Comment:
Please, keep formatting if possible.

Reply:
I am going to add the clang-format config we use in dpctl to numba-dppy. It will help standardize. I will do it in a separate PR.